README.md

```diff
@@ -93,7 +93,6 @@ To test this project just enter this command:
 👤 **Dani Morillo**
 
 - GitHub: [danifromecuador](https://github.com/danifromecuador)
```

Comment:
- [ ] Its understandable you removed the link since LinkedIn has not approved your account yet, sorry about that, you can write to their support team